pkg/openshift/certgen/files.go

```diff
@@ -33,11 +72,32 @@ func (c *Config) PrepareMasterFiles() error {
 
 // WriteMasterFiles writes the templated master config
 func (c *Config) WriteMasterFiles(fs filesystem.Filesystem) error {
-	for _, name := range templates.AssetNames() {
+
+	// create special case directories
+	var specialCaseDirs = map[string]filesystem.Fileinfo{}
+	specialCaseDirs["tmp"] = filesystem.Fileinfo{
+		User:  "root",
+		Group: "root",
+		Mode:  os.FileMode(01777),
+	}
```

Comment:
```
	specialCaseDirs := map[string]filesystem.Fileinfo{
		"tmp": {
			User:  "root",
			Group: "root",
			Mode:  os.FileMode(01777),
		},
		...
	}
```

Reply:
I struggled with this initially when I tried it.  Updated as this is cleaner.